src/backend/joanie/core/migrations/0001_initial.py

```diff
@@ -311,6 +311,7 @@ class Migration(migrations.Migration):
                             ("ml", "Malayalam"),
                             ("mn", "Mongolian"),
                             ("mr", "Marathi"),
+                            ("ms", "Malay"),
```

Comment:
In theory we should not modify migration files anymore because we made an initial release and deployed to production.
In practice your modification does not do anything on the database so let's keep it but keep in mind that migrations are now considered stable.